app/models/sam_account_reckoner.rb

```diff
@@ -31,4 +45,16 @@ def sam_status
   def client
     @client ||= Samwise::Client.new(api_key: DataDotGovCredentials.api_key)
   end
+
+  def vendor_summary
+    @vendor_summary ||= client.get_vendor_summary(duns: user.duns_number)
+  end
+
+  def duns_is_in_sam?
+    vendor_summary[:in_sam]
```

Comment:
Samwise::Client#get_vendor_summary returns a plain Ruby hash (https://github.com/18F/samwise/blob/master/lib/samwise/client.rb#L17-L24) so I think this is fine.